samples/csharp_dotnetcore/13.core-bot/Dialogs/BookingDialog.cs

```diff
@@ -52,12 +53,12 @@ private async Task<DialogTurnResult> OriginStepAsync(WaterfallStepContext stepCo
 
             if (bookingDetails.Origin == null)
             {
-                return await stepContext.PromptAsync(nameof(TextPrompt), new PromptOptions { Prompt = MessageFactory.Text("Where are you traveling from?") }, cancellationToken);
-            }
-            else
-            {
-                return await stepContext.NextAsync(bookingDetails.Origin, cancellationToken);
+                var promptMessage = MessageFactory.Text("Where are you traveling from?", inputHint: InputHints.ExpectingInput);
```

Comment:
shouldn't a prompt be always setting ExpectingInput?
is this a bug in the prompt?